src/data/local/github/tracks-node/track-3/steps/step-2.md

```diff
@@ -0,0 +1,16 @@
+# Step 1 - Integrating with the database
+
+### Estimated time: 4 hours
+
+Now you're going to fully integrate the Login mutation with the database. Since we don't have a mutation to create users (yet), we should create one directly on the database, so we have the conditions to test it properly.
```

Comment:
```suggestion
Now you're going to fully integrate the Login mutation with the database. Since we don't have a mutation to create users (yet), we should create one directly on the database, so we have some conditions to test it properly.
```